scripts/ci/trigger_docs_update

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Trigger the job that will update the documentation website.
+# Argument:
+#   version: the version of the new release.
+
+set -ev
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: $0 <version>"
+    exit 1
+fi
+
+readonly VERSION="$1"
+
+readonly BODY="{
+    \"request\": {
+        \"branch\": \"master\",
+        \"config\": {
+            \"env\": [\"VERSION="${VERSION}"\"]
```

Comment:
There should be no quotes around `${VERSION}`.

Reply:
Hum, for some reason this gets nice coloring on emacs with the extra quotes. But I'm not sure now where I got this style from :thinking: I was trying to follow google's style guide, and it seems they recommend to remove the quotes. So off they go.